src/changelog/2023/10/blueprints.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Decide to remove the `changelog` tag from the other one, or keep these. I've reviewed this one now anyway.